api/golang/core/lib/enclaves/enclave_context.go

```diff
@@ -47,8 +47,22 @@ const (
 	osPathSeparatorString = string(os.PathSeparator)
 
 	dotRelativePathIndicatorString = "."
+
+	// required to get around "only Github URLs" validation
+	composePackageIdPlaceholder = "github.com/NOTIONAL_USER/USER_UPLOADED_COMPOSE_PACKAGE"
 )
 
+// TODO Remove this once package ID is detected ONLY the APIC side (i.e. the CLI doesn't need to tell the APIC what package ID it's using)
+// Doing so requires that we upload completely anonymous packages to the APIC, and it figures things out from there
+var supportedDockerComposeYmlFilenames = []string{
```

Comment:
this list exists twice in  `api` and once in `core` but idk a great place to put a `const` for this so it's defined only once, maybe create a `kurtosis_constants` library?